internal/kallsyms/kallsyms_test.go

```diff
@@ -38,6 +40,9 @@ func TestParseSyms(t *testing.T) {
 
 func TestParseProcKallsyms(t *testing.T) {
 	f, err := os.Open("/proc/kallsyms")
+	if runtime.GOOS != "linux" && errors.Is(err, os.ErrNotExist) {
```

Comment:
Same story here, I'd mark Linux-only tests with `testutils.Linux(t)` or something similar to keep things expressive. You're being extra defensive here making sure /proc/kallsyms doesn't exist, but I don't see how this would ever run on Windows, and afaik macOS doesn't have kallsyms either.

Reply:
Yeah I meant to remove this, but somehow it slipped through.